tools/ci_build/build.py

```diff
@@ -604,10 +604,14 @@ def get_config_build_dir(build_dir, config):
 
 
 def run_subprocess(args, cwd=None, capture_stdout=False, dll_path=None,
-                   shell=False, env={}, python_path=None):
+                   shell=False, env={}, python_path=None, add_dll_dir_path=None):
     if isinstance(args, str):
         raise ValueError("args should be a sequence of strings, not a string")
 
+    # print("Added DLL directory everytime:", add_dll_dir_path)
+    if add_dll_dir_path and is_windows():
+        os.add_dll_directory(add_dll_dir_path)
```

Comment:
Please add some comments. Who need this? Why it is needed?

Reply:
Part of testing, removed.